manual/installation.tex

```diff
@@ -175,10 +179,13 @@ \subsection{Environment variables}
 A number of environmental variables affect the build.  In particular
 they can control the default paths for libraries, the default
 compilers, etc.  The list of environmental variables is given below:
+\verbatimfont{\footnotesize}%
 \begin{verbatim}
 CXX              C++ compiler
 CC               C Compiler
 MKL_HOME         Path for MKL
+MKL_ROOT
```

Comment:
Sorry. I was giving the wrong info.
Please list only MKL_HOME for environment variable.
When we tell users, just give them 1 but our setting can be smarter internally.

Reply:
Good that will be less confusing